### DISCUSSION TITLE
New user - how to just view someone's project

### DISCUSSION THREAD

**Discussion by DavePanetta** [community/community#54033] [New to GitHub] [answered]
I just want to view someone elses project, I thought the desktop would make it simpler but all the directions just tell me how to create my own repository - what do I not understand?

Everything I see is about creating and codeing, how does a new guy just browse?

Perhaps my wording for a search was wrong

**Comment by D2Machine** [comment, marked as answer]
Hi Welcome to GitHub 👋🏻 
You can see others repositories on GitHub explore : https://github.com/explore
However, if you don't see anything in the search result, your advanced search may change. Make sure it is clear.
Some search tips : https://www.freecodecamp.org/news/github-search-tips/
[GitHub Documant](https://docs.github.com/en)
If this answer was useful [Mark It](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer)

**Reply by DavePanetta** [reply]
Thanks, that will get me going.
